buildifier/utils/utils.go

```diff
@@ -68,3 +72,47 @@ func GetParser(inputType string) func(filename string, data []byte) (*build.File
 		return build.ParseDefault
 	}
 }
+
+// WriteTemp writes data to a temporary file and returns the name of the file.
+func WriteTemp(data []byte, toRemove *[]string) (file string, err error) {
+	f, err := ioutil.TempFile("", "buildifier-tmp-")
+	if err != nil {
+		return "", fmt.Errorf("creating temporary file: %v", err)
+	}
+	name := f.Name()
+	*toRemove = append(*toRemove, name)
+	defer f.Close()
+	_, err = f.Write(data)
```

Comment:
push into if, use :=

Reply:
Done (I've just copied the code here, but good to refactor it at the same time)